### DISCUSSION TITLE
blog/release-of-tailwind-nextjs-starter-blog-v2

### DISCUSSION THREAD

**Discussion by giscus** [Mandy9943/techify#3] [Announcements]
# blog/release-of-tailwind-nextjs-starter-blog-v2

Release of Tailwind Nextjs Starter Blog template v2.0, refactored with Nextjs App directory and React Server Components setup.Discover the new features and how to migrate from V1.

http://localhost:3000/blog/release-of-tailwind-nextjs-starter-blog-v2.0

**Comment by Mandy9943** [comment]
zxczxcx

**Comment by Mandy9943** [comment]
6325+6
